lib/credly/actions/organizations.rb

```diff
@@ -7,8 +7,24 @@ def organizations_list
         connection.get('organizations')
       end
 
-      def organization_get(id)
-        connection.get("organizations/#{id}")
+      def organization_get(organization_id)
+        connection.get("organizations/#{organization_id}")
+      end
+
+      def organization_update(organization_id, body_params)
+        connection.put("organizations/#{organization_id}", body_params)
+      end
+
+      def organization_event_get(organization_id, event_id)
+        connection.get("organizations/#{organization_id}/events/#{event_id}")
+      end
+
+      def organization_tokens_list(organization_id)
+        connection.get("organizations/#{organization_id}/authorization_tokens")
+      end
+
+      def organization_event_list(organization_id, page = {})
```

Comment:
Instead of `page`, I'd probably just stick to your naming convention of `query_params` here. If Credly ever adds other other query params to this endpoint then we don't need to worry about things on our end.